core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -111,7 +111,13 @@ export class ServiceDetailComponent implements OnInit {
         slackChannel: service.slack_channel,
         repository: service.repository,
         tags: service.tags,
-        endpoints: service.endpoints
+        endpoints: service.endpoints,
+        is_public_endpoint:service.is_public_endpoint,
+        create_cloudfront_url:service.metadata.create_cloudfront_url,
+        eventScheduleRate:service.metadata.eventScheduleRate,
+        event_source:service.metadata.event_source_dynamodb,
+        app_name:service.app_name,
```

Comment:
No app_name in service metadata for jazz opensource.

Reply:
Removed app_name in the new pr (PR 549)